internal/cluster/refresh_status_preprocessor.go

```diff
@@ -99,6 +133,17 @@ func (r *refreshPreprocessor) preprocess(ctx context.Context, c *clusterPreproce
 	for _, validationResults := range validationsOutput {
 		sortByValidationResultID(validationResults)
 	}
+
+	// Now that all validations have been executed, skip the validations that we wish to ignore.
+	for _, currentResult := range validationsOutput {
+		for _, v := range currentResult {
+			if shouldIgnoreValidation(ignoredValidations, string(v.ID)) && !stateMachineInput[string(v.ID)] {
```

Comment:
Why there is a need for !stateMachineInput[string(v.ID)] ? Wouldn't it runs only once per validation?